src/System.Net.Security/src/System/Net/Security/Sslstream.Implementation.cs

```diff
@@ -151,12 +151,8 @@ private void ValidateCreateContext(SslAuthenticationOptions sslAuthenticationOpt
 
         private object SyncLock => _context;
 
-        private X509Certificate InternalLocalCertificate => _context.IsServer ? _context.LocalServerCertificate : _context.LocalClientCertificate;
-
         private bool IsShutdown => _shutdown;
```

Comment:
Can this one be removed as well, and call sites just use _shutdown?

What about HandshakeCompleted and just use _handshakeCompleted?

Reply:
Yeah I will ditch them now, turns out SslStream was a mash up of property use, Context vs _context was about 50/50 split on using them for instance.

Reply:
Done